Modules/posixmodule.c

```diff
@@ -1734,6 +1734,39 @@ posix_fildes_fd(int fd, int (*func)(int))
 
 
 #ifdef MS_WINDOWS
+static wchar_t *
+win32_wgetcwd(wchar_t *buf, DWORD buf_size)
+{
+    wchar_t *local_buf = buf;
+    while (1) {
+        wchar_t *temp;
+        DWORD result = GetCurrentDirectoryW(buf_size, local_buf);
+        if (!result)
+            goto fail;
```

Comment:
Per PEP 7, all new code requires braces, even where C allows omitting them.

```suggestion
        if (!result) {
            goto fail;
        }
```

Reply:
Ah, sorry about that, for some reason the braces were missing in the original PR too.